### ISSUE TITLE
Feature request: Allow for the manual setting of colors

### ISSUE THREAD

**Issue by brimwats** [kepano/obsidian-advanced-appearance#2]
The current "accent color hue" is set through sliders like this 
![image](https://user-images.githubusercontent.com/49103046/104084329-dc063400-51fa-11eb-8554-166cf7a633cd.png)

This is user-friendly but prevents the inputting of themes via hex/rgb code (or something similar). Perhaps a user-fillable box could be shown that can set the colors? This could update based on the current slider settings so 1) folks can see what hex/rgb color they currently have  and 2) put in the hex/rgb color that they want

(in accessibility circles hex is usually preferred over RGB)

**Comment by kepano** [comment]
A couple issues with this:

1. The plugin relies on using HSL values (not hex or RGB) because they can be manipulated via CSS to generate all of the other colors. 
2. The sliders are designed to limit the gamut to prevent issues with low contrast combinations

The first could be solved via an input field that automatically converts the values from hex to HSL. Not sure about the second.

**Comment by brimwats** [comment]
HSL values should work fine, so long as it's labelled that way!

The use case I'm thinking of is (for example) using this plugin to make base colors gruvbox-ish. I'm not sure if there is a way to prevent issues with low contrast but I feel like they'd be minority of issues? perhaps these two settigs could be in an "advanced" toggle with that warning?

**Comment by NeighNeighNeigh** [comment]
Yeah I love the idea of the Advance Appearance plugin, but the automatic colours are a dealbreaker. I get the appeal of a hue slider and the math adjustments that come afterward. But the resulting palettes are too monotone. It would be nicer imo if the plugin could support easier development of fully hand crafted colour themes.
